cocos/platform/openharmony/OpenHarmonyPlatform.cpp

```diff
@@ -51,6 +53,29 @@ void onSurfaceCreatedCB(OH_NativeXComponent* component, void* window) {
     sendMsgToWorker(cocos2d::MessageType::WM_XCOMPONENT_SURFACE_CREATED, component, window);
 }
 
+void onSurfaceHideCB(OH_NativeXComponent* component, void* window) {
+    int32_t ret;
+    char idStr[OH_XCOMPONENT_ID_LEN_MAX + 1] = {};
+    uint64_t idSize = OH_XCOMPONENT_ID_LEN_MAX + 1;
+    ret = OH_NativeXComponent_GetXComponentId(component, idStr, &idSize);
+    if(ret != OH_NATIVEXCOMPONENT_RESULT_SUCCESS) {
```

Comment:
The point of getting idStr is to determine if it is valid?

Reply:
yes，confirm whether XCompoonent is valid